operations/runbooks.rst

```diff
@@ -96,3 +159,94 @@ Zápis z jednorázového elektronického hlasování
    Pokud nějaký člen rady nehlasoval, tak není potřeba jej vůbec uvádět.
 
 #. Pošli Pull Request s touto změnou. Před jeho přijetím by měl být schválen alespoň jedním dalším členem výboru (můžeš nastavit tým `@pyvec/board <https://github.com/orgs/pyvec/teams/board>`__ v *reviewers*).
+
+
+.. _navody-pokladnik:
+
+Návody pro pokladníka
+---------------------
+
+.. _proplaceni:
+
+Proplácení účetních dokladů
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++------------------------------+------------------------------------------------------+
+| **Pro koho je tento návod:** | |:moneybag:| :term:`pokladník <Pokladník>`, předseda |
++------------------------------+------------------------------------------------------+
+
+#. Někdo využívá :ref:`finančních služeb Pyvce <financni-sluzby>` a krok za krokem následuje návod :ref:`jak-proplatit`. Na Pyvec přijde účetní doklad k proplacení a e-mail s dalšími detaily:
+
+   - pod jakou **skupinu** výdaj spadá
+   - **co to je** za věc a **k čemu slouží**
+   - na jaké **číslo účtu** mají být proplaceny peníze
+
+#. Přihlásíš se do `Fio banky <https://ib.fio.cz/ib/login>`_ a odešleš platbu, přičemž využiješ detaily z e-mailu:
```

Comment:
Trochu mi tady chybí jakákoliv kontrola: ať už z hlediska chyb, bezpečnosti nebo souhlasu Pyvce.

Podle současného návodu pokladník proplatí všechno, co přijde e-mailem.

Reply:
No jo, to vychází z toho, že teď je pokladník ve své dvojjedinosti i předseda a nejedná jako robot, ale kontrolu dělá automaticky v hlavě. Předseda ví, jestli se něco takového schválilo nebo ne. Přidám větu, že by měl pokladník (což je z definice pokladníka člověk z výboru) ověřit, že toto bylo předtím schváleno nebo diskutováno a že částky souhlasí.

Reply:
4c6b03787e506db44bcd858d638e990a068eee0f